### PR TITLE
Update foo.txt

### PR DESCRIPTION
**Pull request by liamdebeasi**

### CODE REVIEW

File: foo.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 test file
+ 123
```
Comment (rwaskiewicz) [inline foo.txt:2]: @liamdebeasi I believe you have a leading space here. This is a blocker 🧌

**Review by rwaskiewicz — CHANGES_REQUESTED** [review]

File: foo.txt (hunk 1 above)
Comment (liamdebeasi) [reply foo.txt:2]: This white space is critical to the functionality of the system. Without it, the entire thing collapses.

Comment (rwaskiewicz) [reply foo.txt:2]: Can we add a comment then to say this is load bearing? Otherwise, people might have this argument over and over again

Comment (liamdebeasi) [reply foo.txt:2]: The whitespace requirement is pretty obvious just by reading the code. #WontFix

Comment (rwaskiewicz) [reply foo.txt:2]: Hmmmm I’m wondering if you don’t think this is worth fixing. Maybe we should disagree and commit here. I’d hate for this work to get lost

**Review by rwaskiewicz — APPROVED** [review]

File: foo.txt (hunk 1 above)
Comment (liamdebeasi) [reply foo.txt:2]: Excellent idea! I am going to disagree with your suggestion and commit to mine. Thanks in advance.

Comment (rwaskiewicz) [reply foo.txt:2]: Yes, let’s go with your idea. I had hoped
I’d convince you my idea was your idea, but that seems to have failed. Open source, amirite?

**Comment by liamdebeasi** [comment]
Closing as this project is abandoned

**Pull request closed without merging**